benchmarks/mmlu/mmlu_eval.py

```diff
@@ -167,7 +175,7 @@ def main(config):
     subject_total[subject] += 1
 
     if idx % 50 == 0:
-      max_logging.log(f" Accuracy: {correct_count / total_count:.4f}")
+      max_logging.log(f" Accuracy: {correct_count / total_count:.4f} | {total_count}/{len(mmlu_test_ds)}")
```

Comment:
Nit: shall we add something like `processed` along with `{total_count}/{len(mmlu_test_ds)}` or similar to avoid confusion?